aptos-move/vm-genesis/src/lib.rs

```diff
@@ -434,6 +434,7 @@ pub fn default_features() -> Vec<FeatureFlag> {
         FeatureFlag::CRYPTOGRAPHY_ALGEBRA_NATIVES,
         FeatureFlag::BLS12_381_STRUCTURES,
         FeatureFlag::STORAGE_SLOT_METADATA,
+        FeatureFlag::CHARGE_INVARIANT_VIOLATION,
```

Comment:
cc @perryjrandall Let's look into removing these hardcoded values in genesis and favoring a testnet-like upgrade process for devnet